docs/intentReaderHandlers/RunTestHandler.md

```diff
@@ -0,0 +1,338 @@
+                    "..."
+                },
+                "asynchronous": false,
+                "appType": "<appType>"
+            }
+        },
+        "context": {
+            "source": "device"
+        }
+    }
+
+
+<details>
+    <summary> Response of the valid request </summary>
+</details>
+
+    {
+        "stats": {
+            "pending": 0,
+            "pendingPercent": 0,
+            "other": 0,
+            "hasOther": false,
+            "skipped": 13,
+            "hasSkipped": true,
+            "suites": 1,
+            "duration": 93015,
+            "tests": 199,
+            "start": "<startTime>",
+            "end": "<endTime>",
+            "testsRegistered": 199,
+            "passes": 178,
+            "failures": 8,
+            "passPercent": 89
+        },
+        "results": [
+            {
+                "uuid": "<uuid>",
+                "title": "Summary",
+                "fullFile": "",
+                "file": "",
+                "beforeHooks": [],
+                "afterHooks": [],
+                "tests": [],
+                "suites": [
+                    {
+                        "uuid": "<uuid>",
+                        "title": "Summary",
+                        "fullFile": "",
+                        "file": "",
+                        "beforeHooks": [],
+                        "afterHooks": [],
+                        "tests": [],
+                        "suites": [],
+                        "passes": [],
+                        "failures": [],
+                        "pending": [],
+                        "skipped": [],
+                        "duration": 0,
+                        "root": false,
+                        "rootEmpty": false,
+                        "_timeout": 2000
+                    }
+                ],
+                "passes": [],
+                "failures": [],
+                "pending": [],
+                "skipped": [],
+                "duration": 0,
+                "root": true,
+                "rootEmpty": true,
+                "_timeout": 2000
+            },
+            {
+                "uuid": "<uuid>",
+                "title": "Firebolt SDK Version : "<version>" , Mode : "<mode>" , Firmware : <firware> , Hash : <hash> , AppId : <appId>",
+                "fullFile": "",
+                "file": "",
+                "beforeHooks": [],
+                "afterHooks": [],
+                "tests": [],
+                "suites": [
+                    {
+                        "uuid": "<uuid>",
+                        "title": "CORE",
+                        "fullFile": "",
+                        "file": "",
+                        "beforeHooks": [],
+                        "afterHooks": [],
+                        "tests": []
+                    }
+                ]
+            }
+        ]
+    }
+
+----------------------------------------------------------------------------------------------------------------------
+
+
```

Comment:
Missing the header to signify invalid request/responses starting.

Reply:
added the header